From e187f5dca8e844898234ec37bd49e01b7f7845de Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 23 Jul 2024 12:12:37 +0300
Subject: [PATCH] [SW-191599] g3: add status prints

Due to unexplained and non-reproducible stuck condition in
hl_sram_self_repair(), additional status prints are added,
so next time it will be easier to pinpoint the issue

Change-Id: I258928fed2b92939ea1dfc81a133959eaa18e3c4
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/asic.c b/arch/gaudi3/arc/asic.c
index 0272d87..fbe0858 100644
--- a/arch/gaudi3/arc/asic.c
+++ b/arch/gaudi3/arc/asic.c
@@ -62,6 +62,8 @@
 	if (boot_info_set_no_merge(HL_BOOT_INFO_SRAM_BIST_RESULT, (void *)&bist_data,
 				 sizeof(struct hl_ras_sram_bist_data)))
 		preboot_puts("BOOT_INFO: Failed to set sram mbist result\n");
+	else
+		preboot_puts("BOOT_INFO: sram mbist result stored\n");
 }
 
 static bool ppboot_try_sram_mbist_data_img(struct boot_desc *desc)
@@ -188,6 +190,8 @@
 	gaudi3_mem_init(HL_SRAM_BASE_FULL, HL_SRAM_SINGLE_DIE_SIZE);
 	gaudi3_clear_sram_ecc();
 	gaudi3_mask_sram_ecc(false);
+
+	preboot_puts("sram ecc init done\n");
 }
 
 void hl_sram_self_repair(void)
